libraries/deeplink/impl/src/main/kotlin/io/element/android/libraries/deeplink/impl/DefaultDeepLinkCreator.kt

```diff
@@ -27,6 +28,15 @@ class DefaultDeepLinkCreator : DeepLinkCreator {
                     append("/")
                     append(threadId.value)
                 }
+
+                if (eventId != null) {
+                    if (threadId == null) {
+                        // Add extra separator to mark the thread slot as empty
+                        append("/")
+                    }
+                    append("/")
+                    append(eventId.value)
+                }
```

Comment:
It works, but maybe we could simplify a bit the complexity and have:
```kotlin
        return buildString {
            append("$SCHEME://$HOST/")
            append(sessionId.value)
            append("/")
            append(roomId?.value.orEmpty())
            append("/")
            append(threadId?.value.orEmpty())
            append("/")
            append(eventId?.value.orEmpty())
        }
```
at the price of having less nice links :)